MagazineLayout/Public/MagazineLayout.swift

```diff
@@ -407,9 +392,9 @@ public final class MagazineLayout: UICollectionViewLayout {
         withID: movedItemID,
         .beforeUpdates)
     {
-      return previousLayoutAttributesForItem(at: initialIndexPath)?.copy() as? UICollectionViewLayoutAttributes
+      return previousLayoutAttributesForItem(at: initialIndexPath)
     } else {
-      return nil
+      return super.layoutAttributesForItem(at: itemIndexPath)
```

Comment:
Thinking it's appropriate to return super instead of nil. I haven't seen this code get hit before so I suspect this  is effectively a no-op, unless collection view behavior changes.